### DISCUSSION TITLE
Component Space Simulation

### DISCUSSION THREAD

**Discussion by KratosBoy123** [pafuhana1213/KawaiiPhysics#123] [Ideas - 要望・アイデア]
I cannot seem to be able to find the choice to change the simulation space for a Kawaii Physics node in the Animation Blueprint.
A Kawaii physics node seems to be operating always in World Space.

Is this a feature that will come soon or something already available?

Thanks

**Comment by pafuhana1213** [comment]
Hi @Federico345 

Setting the World Damping property to 0 should complete the process in ComponentSpace, but does this meet your expectations?
![image](https://github.com/pafuhana1213/KawaiiPhysics/assets/8957600/a7636a2f-7c83-4ee2-bfcb-96260fc2dc81)

If you don't meet the requirements, it would be helpful if you could provide a specific example.

Thanks!

**Reply by KratosBoy123** [reply]
I actually did it by applying a value of 10 to the World Damping Rotation and Location instead of 0. The situation is that I have a character blueprint (with a skeletal mesh component inside) attached to another object and this one moves in the world. I need however my attached skeletal mesh component to simulate Kawaii while being attached.

Setting these values to 10 worked perfectly and I achieved what I needed. Thanks so much for the kind help!
